src/privateDogu.html

```diff
@@ -0,0 +1,618 @@
+            <p>The above copyright notice and this permission notice shall be included in all copies or substantial portions of
+              the Software. </p>
+              <p>THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING
+              BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+              NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+              DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+              FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+              SOFTWARE.</p>BlueSpice Plugin Lizenzen: <a target="_blank"
+                                                         href="https://de.wiki.bluespice.com/wiki/Setup:Softwarekatalog">https://de.wiki.bluespice.com/wiki/Setup:Softwarekatalog</a>.
+            </span>
+            </div>
+          </div>
+        </div>
+        <div class="accordion-item panel-default">
+        <div class="accordion-header panel-default panel-heading" id="openldap_panel">
+          OpenLDAP Dogu
+        </div>
+        <div align="justify" class="accordion-content panel-default panel-body">
+            <span>
+              <p>Copyright © 2024 Cloudogu GmbH</p>
+
+              <p>Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated
+                documentation files (the "Software"), to deal in the Software without restriction, including without limitation
+                the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and
+                to permit persons to whom the Software is furnished to do so, subject to the following conditions: </p>
+              <p>The above copyright notice and this permission notice shall be included in all copies or substantial portions of
+                the Software. </p>THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING
+                BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+                NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+                DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+                FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+                SOFTWARE.
+              </span>
+        </div>
+      </div>
+      <div class="accordion-item panel-default">
+        <div class="accordion-header panel-default panel-heading" id="redmine_panel">
+          Redmine Dogu
+        </div>
+        <div align="justify" class="accordion-content panel-default panel-body">
+            <span>
+              <p>MIT License</p>
```

Comment:
Uh, the Cloudogu source of the Redmine dogu is actually licensed as AGPL 3.0-only too. What does this mean then?

Reply:
On the same note what does this public dogu have to do in a file called private dogu? This file is full of public dogus :thinking:

Reply:
We can rename this into thirdParty instead of private. Poor choice of words maybe.